lib/ukatomic/Config.uk

```diff
@@ -0,0 +1,3 @@
+menuconfig LIBUKATOMIC
+	bool "ukatomic: Atomic operations"
+	default y
```

Comment:
A `help` section with some additional info on the contents of this library would be useful.